master/security/service-accounts.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Use service accounts in policy rules
+---
+
+### Big picture
+
+Use Calico network policy to allow/deny traffic for Kubernetes services. 
+
+### Value
+
+Using Calico network policy, you can leverage Kubernetes service accounts with RBAC for flexible control over how policies are applied in a cluster. For example, the Network Security can have RBAC permissions to:
+
+- Control which service accounts the Developer team can use within a namespace
+- Write high-priority network policies for those service accounts (that the Developer team cannot override) 
+
+The Network Security team can maintain full control of security, while selectively allowing Developer operations where it makes sense.  
```

Comment:
I think "Network", "Security" and "Developer" should be lowercase in this sentence

Reply:
(and a couple of other places in the doc)

Reply:
Fixed.

Reply:
Fixed.